src/components/fields/SymbolSelector.js

```diff
@@ -353,27 +354,38 @@ const SYMBOLS = [
 ];
 
 class SymbolSelector extends Component {
-  constructor(props) {
-    super(props);
-    this.setLocals(props);
+  constructor(props, context) {
+    super(props, context);
+    this.setLocals(props, context);
   }
 
-  componentWillReceiveProps(nextProps) {
-    this.setLocals(nextProps);
+  componentWillReceiveProps(nextProps, nextContext) {
+    this.setLocals(nextProps, nextContext);
   }
 
-  setLocals(props) {
+  setLocals(props, context) {
     const {fullContainer} = props;
+    const {defaultContainer} = context;
 
     this.markerColor = nestedProperty(fullContainer, 'marker.color').get();
     this.borderWidth = nestedProperty(fullContainer, 'marker.line.width').get();
 
+    if (this.markerColor === MULTI_VALUED) {
```

Comment:
this is very tricky: we don't actually end up using default values here and I couldn't figure out how to do it... we use the values from the first trace. I've added a note to fit'n'finish

Reply:
well, that's what the current workspace does, sounds ok to me!